<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrDialog</class>
 <widget class="QDialog" name="RegistrDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>560</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="11,2">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="Upper_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,4,10">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer_1">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;fwfew fwfwe fwfw&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel::toolTip {
	background-color: blue;	
	color: green;
}</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="login_page">
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>421</width>
            <height>181</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,2,3,2">
           <property name="spacing">
            <number>7</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="labelEmail_1">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Email</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditEmail_1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>IBeamCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="maxLength">
              <number>30</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelPassword_1">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Password</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditPassword_1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="maxLength">
              <number>30</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="incorrectData">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>226</y>
            <width>201</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
         <zorder>incorrectData</zorder>
         <zorder>layoutWidget</zorder>
        </widget>
        <widget class="QWidget" name="create_page">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>441</width>
            <height>291</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,1,2">
           <property name="horizontalSpacing">
            <number>15</number>
           </property>
           <property name="verticalSpacing">
            <number>7</number>
           </property>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="lineEditName">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="inputMethodHints">
              <set>Qt::InputMethodHint::ImhNone</set>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="maxLength">
              <number>20</number>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="2" colspan="2">
            <widget class="QLineEdit" name="lineEditSurname">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>11111111</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maxLength">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="4">
            <widget class="QLineEdit" name="lineEditEmail_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>500</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maxLength">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="4">
            <widget class="QLineEdit" name="lineEditPassword_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>500</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maxLength">
              <number>30</number>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="4">
            <widget class="QLineEdit" name="lineEditNumber">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>500</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maxLength">
              <number>20</number>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="labelName">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item row="0" column="2" colspan="2">
            <widget class="QLabel" name="labelSurname">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Surname</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="4">
            <widget class="QLabel" name="labelPhone">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Phone number</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item row="6" column="3">
            <widget class="QLabel" name="incorrectPassword">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing</set>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="3">
            <widget class="QLabel" name="labelPassword_2">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Password</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="3">
            <widget class="QLabel" name="labelEmail_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Email</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="incorrectEmail">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="exclamationMark">
          <property name="geometry">
           <rect>
            <x>425</x>
            <y>294</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Password must contest:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Not less than 8 characters;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;A lowercase and uppercase letter;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;At least one digit;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Spesial symbol (!, @, #, $ ,%, ^, &amp;amp;, *, -, _,);&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Not more than 30 characters;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolTip {
    background-color:  #212121;
    color: #D3D3D3;
    border: 1px solid#212121;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>C:/Users/golov/Downloads/exclamationmark_white.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="lower_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QPushButton" name="acceptButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>15</y>
        <width>121</width>
        <height>45</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>SIGN IN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changeButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>22</y>
        <width>151</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>CREATE ACCOUNT</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
